internal/init_test.go

```diff
@@ -154,7 +154,7 @@ func skipf(t *testing.T) func(format string, args ...any) {
 // detects if test is run from "debug test" feature in VSCode
 func isInDebug() bool {
 	ex, _ := os.Executable()
-	return path.Base(ex) == "__debug_bin"
+	return strings.HasPrefix(path.Base(ex), "__debug_bin")
```

Comment:
Small bugfix to support newest version of Delve (see https://github.com/databricks/terraform-provider-databricks/pull/2497)